integration-cli/docker_cli_ps_test.go

```diff
@@ -508,3 +508,22 @@ func (s *DockerSuite) TestPsListContainersFilterCreated(c *check.C) {
 		c.Fatalf("Expected id %s, got %s for filter, out: %s", cID, containerOut, out)
 	}
 }
+
+func (s *DockerSuite) TestPsFormatMultiNames(c *check.C) {
```

Comment:
I've updated the test to do both given we already have linked containers running in the same test